fontforge/parsettf.c

```diff
@@ -782,7 +782,7 @@ return;
 
     for ( i=0; i<info->numtables; ++i ) {
 	tabs[i].tag = getlong(ttf);
-	tabs[i].checksum = getlong(ttf);
+	tabs[i].checksum = getuint32(ttf);
```

Comment:
This was already being silently converted to `uint32`, so that's why I don't also change the struct `tt_tables`.